x-pack/plugins/snapshot_restore/common/types/snapshot.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export interface Snapshot {
+  id: string;
+  summary: SnapshotSummary;
+  repositories: string[];
+}
+
+export interface SnapshotSummary {
```

Comment:
Can we use camelCase in all our client code? I see it much easier if we know in advance that everything is camelCase, regardless of what ES sends.

Reply:
Sorry I missed that this was in "common" and was used on the server 😊